js/index.js

```diff
@@ -26,9 +26,9 @@ const requestPermissions = (background) => {
   NativeModules.RNRadar.requestPermissions(background);
 };
 
-const startTracking = () => {
-  NativeModules.RNRadar.startTracking();
-};
+const startTracking = (options) => {
+  NativeModules.RNRadar.startTracking(options);
+}
```

Comment:
I would keep the semicolon here for consistency. We may want to add linting at some point.